docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -45,7 +45,7 @@ To download the manifest file, run:
 
 ["source", "sh", subs="attributes"]
 ------------------------------------------------
-curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/kubernetes/elastic-agent-managed-kubernetes.yaml
+curl -L -O https://raw.githubusercontent.com/elastic/elastic-agent/{branch}/deploy/kubernetes/elastic-agent-managed/elastic-agent-managed-daemonset.yaml
```

Comment:
@dedemorton , the file name should be the same as previous `elastic-agent-managed-kubernetes.yaml`, only the path from `beats` to `elastic-agent` changes. @ChrsMark can you confirm?

Reply:
yeap, you need the actual file path here.